### DISCUSSION TITLE
Is it possible to create an API key for specific users and set the page update history accordingly?

### DISCUSSION THREAD

**Discussion by phochste** [requarks/wiki#7332] [Help / Questions]
For a project I require API access to a Wiki.js instance where a specific user account would be able to update the text of a page. When a page is updated via the API I would like to see that reflected in the page history. I want to see: `Last edited by user`.

Whatever API token I currently try to create and page updates via the API, the history always shpws `Last edited by Administrator`.  

Is it possible in Wiki.js to set the history to the actual user for which the API was created?

**Comment by jorgementx** [comment]
@phochste did you find a solution to this?

**Reply by phochste** [reply]
@jorgementx No, this was for a demonstration project. I faked the specific user account by editing some of the Wiki.js templates.

**Comment by juliantcook** [comment]
It would be great to have this now for agentic workflows.
